core/src/main/scala/org/http4s/package.scala

```diff
@@ -100,4 +100,11 @@ package object http4s { // scalastyle:ignore
       case Left(l) => f(l)
     }
   }
+
+  final implicit class Http4sLiteralSyntax(val sc: StringContext) extends AnyVal {
```

Comment:
Is this "syntax"?  Should it be imported through `org.http4s.implicits._` rather than be part of the package object?  My gut feeling is yes, but ip4s does it at the top level.

@mpilquist, is this a design you considered and rejected?